pkg/gci/gci.go

```diff
@@ -154,56 +162,54 @@ func LoadFormatGoFile(file io.FileObj, cfg config.Config) (src, dist []byte, err
 	firstWithIndex := true
 
 	var body []byte
-	// order: standard > default > custom
-	if len(result["standard"]) > 0 {
-		for _, d := range result["standard"] {
-			AddIndent(&body, &firstWithIndex)
-			body = append(body, src[d.Start:d.End]...)
-		}
-
-		body = append(body, utils.Linebreak)
-	}
 
-	if len(result["default"]) > 0 {
-		for _, d := range result["default"] {
-			AddIndent(&body, &firstWithIndex)
-			body = append(body, src[d.Start:d.End]...)
-		}
+	// if default order sorted sections
+	if !cfg.CustomOrder {
```

Comment:
I notice that we will sort it for each file which is unnecessary. Please sort it when loading config.

Reply:
Agree with you. Done